iot_verifier/src/poc.rs

```diff
@@ -220,11 +232,13 @@ impl Poc {
                     return Ok(VerifyWitnessResult::not_asserted(InvalidParticipantSide::Beaconer))
                 };
 
-                if let Some(hex_scale) = hex_density_map.get(beaconer_location).await {
-                    Ok(VerifyWitnessResult::valid(witness_info, hex_scale))
+                let tx_scale = if let Some(hex_scale) = hex_density_map.get(beaconer_location).await
```

Comment:
similar to previous instance of setting tx_scale, could use a unwrap_or in place of the if else